docs/lib/auth/fragments/js/social.md

```diff
@@ -68,7 +68,7 @@ Amplify.configure(updatedAwsConfig);
 
 **React Native - Redirect URIs**
 
-For React Native applications, You need to define a custom URL scheme for your application before testing locally or publishing to the app store. This is different for Expo or vanilla React Native. Follow the steps below for React Native iOS & Android or [Expo Linking docs](https://docs.expo.io/versions/latest/workflow/linking/) for more information. After completing those steps, assuming you are using `myapp` as the name of your URL Scheme (or whatever friendly name you have chosen), you will use these URLs as *Sign in Redirect URI(s)* and/or *Sign out redirect URI(s)* inputs. Your URIs could look like any of these:
+For React Native applications, you need to define a custom URL scheme for your application before testing locally or publishing to the app store. This is different for Expo or vanilla React Native. For Expo, first follow the [Expo Linking Guide](https://docs.expo.io/guides/linking/). For React Native, first follow the [React Native Linking Guide](https://reactnative.dev/docs/linking). After completing those steps, assuming you are using `myapp` as the name of your URL Scheme (or whatever friendly name you have chosen), you will use these URLs as *Sign in Redirect URI(s)* and/or *Sign out redirect URI(s)* inputs. Your URIs could look like any of these:
```

Comment:
Minor: to be a bit more explicit about vanilla React Native can we update:
```md
For Expo, first follow the [Expo Linking Guide](https://docs.expo.io/guides/linking/). For React Native, first follow the [React Native Linking Guide](https://reactnative.dev/docs/linking).
```
to:
```md
For Expo, follow the steps in [Expo Linking Guide](https://docs.expo.io/guides/linking/). For vanilla React Native, follow the steps in [React Native Linking Guide](https://reactnative.dev/docs/linking).
```